packages/source-gdocs-paste/test/atjson-source-gdocs-paste-test.ts

```diff
@@ -0,0 +1,108 @@
+import Document from '@atjson/document';
+import { HIR } from '@atjson/hir';
+import KIXSource from '@atjson/source-gdocs-paste';
+import * as fs from 'fs';
+import * as path from 'path';
+
+describe('@atjson/source-gdocs-paste', () => {
+  describe('relatively complex document', () => {
+    var atjson;
+
+    beforeAll(() => {
+      // https://docs.google.com/document/d/1xP_M2SchJt81ZuivsO7oix8Q_fCx4PENKJFJR5npFNM/edit
```

Comment:
This URL is private 😅

Reply:
Updated permissions - I can't share outside Conde, unfortunately. :(